tox.ini

```diff
@@ -351,7 +351,7 @@ deps =
     -r{toxinidir}/requirements.txt
     func-s_base-py{35,36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb==0.7.31
+    git+https://GIThub.com/wandb/yea-wandb.git@jq1/rename-queries#egg=yea-wandb
```

Comment:
we don't want this right?

Reply:
Yep, this was just required until the review.. thanks will merge now with new yea-wandb release